server/form-pages/utils/decorators/page.decorator.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
My preference would be for this to go by the specific lines we want these to apply to

Reply:
Yeah, good call 👍